assets/src/components/ImageOrButton/ImageOrButton.js

```diff
@@ -11,7 +11,7 @@ export class ImageOrButton extends Component {
     const {__} = wp.i18n;
 
     if ( typeof this.props.disabled == 'undefined' ) {
-      this.props.disabled = false;
+      // this.props.disabled = false;
```

Comment:
Is this a mistake...? If not, this entire `if` statement can be removed 😄

Reply:
Hmm I think it was temporary, I had an error message saying that this property couldn't be set because it was readonly, I need to check that again

Reply:
Right, `can't define property "disabled": Object is not extensible`, gonna look for a better fix.